test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -75,22 +75,21 @@ public void VerifyImages(ImageDescriptor imageDescriptor)
         private void VerifySdkImage_NewRestoreRun(ImageDescriptor imageDescriptor, string appSdkImage)
         {
             // dotnet new, restore, build a new app using the sdk image
-            List<string> args = new List<string>();
-            args.Add($"netcoreapp_version={imageDescriptor.DotNetCoreVersion}");
+            List<string> buildArgs = new List<string>();
+            buildArgs.Add($"netcoreapp_version={imageDescriptor.DotNetCoreVersion}");
             if (!imageDescriptor.SdkVersion.StartsWith("1."))
             {
-                args.Add($"optional_new_args=--no-restore");
+                buildArgs.Add($"optional_new_args=--no-restore");
             }
 
-            string buildArgs = GetBuildArgs(args.ToArray());
             string sdkImage = GetDotNetImage(
                 imageDescriptor.SdkVersion, DotNetImageType.SDK, imageDescriptor.OsVariant);
 
             DockerHelper.Build(
                 dockerfile: $"Dockerfile.{DockerHelper.DockerOS.ToLower()}.testapp",
-                fromImage: sdkImage,
                 tag: appSdkImage,
-                buildArgs: buildArgs);
+                fromImage: sdkImage,
+                buildArgs: buildArgs.ToArray());
```

Comment:
`ToArray()` is that better (size/time) as opposed to passing the list.

Reply:
buildArgs is a params therefore it has to be an array